libguard/guard_interface.cpp

```diff
@@ -183,7 +183,8 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
     {
         if (existGuard.targetId == entityPath)
         {
-            if (existGuard.errType == GARD_Reconfig)
+            if ((existGuard.errType == GARD_Reconfig) ||
+                (existGuard.recordId == GUARD_RESOLVED))
```

Comment:
Must be a new record right? Why overriding the existing record? How you will get the resolved record history if you override the existing record?

Reply:
Done. As discussed and agreed creating new guard record